core/dgs-prepare.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/python3
+
+import os, re, datetime, argparse, shutil, json, colorama
+from colorama import Fore as cf
+
+colorama.init()
+
+VERSION = "1.00"
```

Comment:
Každý podscript má vlastnú verziu? Je vôbec dôležité vypisovať verziu pri spustení scriptu?